src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/Namespace/SetAzureEventHubNamespace.cs

```diff
@@ -23,126 +23,57 @@ namespace Microsoft.Azure.Commands.EventHub.Commands.Namespace
+        [Parameter(Mandatory = true, ParameterSetName = AutoInflateParameterSet, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name.")]
         [ResourceGroupCompleter]
         [ValidateNotNullOrEmpty]
          public string ResourceGroupName { get; set; }
 
-        [Parameter(Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            Position = 1,
-            HelpMessage = "EventHub Namespace Name.",
-          ParameterSetName = NamespaceParameterSet)]
-        [Parameter(Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            Position = 1,
-            HelpMessage = "EventHub Namespace Name.",
-          ParameterSetName = AutoInflateParameterSet)]
+        [Parameter(Mandatory = true, ParameterSetName = NamespaceParameterSet, ValueFromPipelineByPropertyName = true, Position = 1, HelpMessage = "EventHub Namespace Name.")]
+        [Parameter(Mandatory = true, ParameterSetName = AutoInflateParameterSet, ValueFromPipelineByPropertyName = true, Position = 1, HelpMessage = "EventHub Namespace Name.")]
         [ValidateNotNullOrEmpty]
         [Alias(AliasNamespaceName)]
         public string Name { get; set; }
 
-        [Parameter(Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            Position = 2,
-            HelpMessage = "EventHub Namespace Location.",
-          ParameterSetName = NamespaceParameterSet)]
-        [Parameter(Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            Position = 2,
-            HelpMessage = "EventHub Namespace Location.",
-          ParameterSetName = AutoInflateParameterSet)]
+        [Parameter(Mandatory = true, ParameterSetName = NamespaceParameterSet, ValueFromPipelineByPropertyName = true, Position = 2, HelpMessage = "EventHub Namespace Location.")]
+        [Parameter(Mandatory = true, ParameterSetName = AutoInflateParameterSet, ValueFromPipelineByPropertyName = true, Position = 2, HelpMessage = "EventHub Namespace Location.")]
         [LocationCompleter("Microsoft.EventHub/namespaces")]
         [ValidateNotNullOrEmpty]
         public string Location { get; set; }
 
-        [Parameter(
-          Mandatory = false,
-          Position = 3,          
-          ValueFromPipelineByPropertyName = true,
-          HelpMessage = "Namespace Sku Name.",
-          ParameterSetName = NamespaceParameterSet)]
-        [Parameter(
-          Mandatory = false,
-          Position = 3,
-          ValueFromPipelineByPropertyName = true,
-          HelpMessage = "Namespace Sku Name.",
-          ParameterSetName = AutoInflateParameterSet)]
+        [Parameter(Mandatory = false, ParameterSetName = NamespaceParameterSet, ValueFromPipelineByPropertyName = true, HelpMessage = "Namespace Sku Name.")]
+        [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, ValueFromPipelineByPropertyName = true, HelpMessage = "Namespace Sku Name.")]
```

Comment:
Add position back for both parameter attributes

Reply:
added back